Signal/src/util/OWSBackup.m

```diff
@@ -107,31 +106,6 @@ - (void)setup
     });
 }
 
-- (void)setBackupPrivateKey:(NSData *)value
```

Comment:
Until we have a real backup encryption key, I've returned this and stubbed out the value.